### PR TITLE
deprecate the gender property from the protocol entirely

### PR DESCRIPTION
**Pull request by ebbit1q**
this removes all mentions of it and leaves some comments to reserve those places

### CODE REVIEW

File: servatrice/servatrice.sql [hunk 1]
```diff
@@ -28,6 +28,7 @@ CREATE TABLE IF NOT EXISTS `cockatrice_users` (
   `admin` tinyint(1) NOT NULL,
   `name` varchar(35) NOT NULL,
   `realname` varchar(255) NOT NULL,
+  -- this field is deprecated and will not be used, this could be removed in a database migration
   `gender` char(1) NOT NULL,
```
Comment (ZeldaZach) [inline servatrice/servatrice.sql:32]: we'll want to remove it here, bump db version, and create migrate script in this pr

File: common/pb/session_commands.proto [hunk 2]
```diff
@@ -148,7 +147,7 @@ message Command_AccountEdit {
     }
     optional string real_name = 1;
     optional string email = 2;
-    optional ServerInfo_User.Gender gender = 3;
+    // gender = 3; // deprecated
```
Comment (ZeldaZach) [inline common/pb/session_commands.proto:150]: removed, not deprecated

File: servatrice/servatrice.sql (hunk 1 above)
Comment (ebbit1q) [reply servatrice/servatrice.sql:32]: I'm not sure how to do a db migration, it's some sql script?

Comment (ZeldaZach) [reply servatrice/servatrice.sql:32]: Correct, we have a folder of them within the repo already

**Review by ZeldaZach — APPROVED** [review]
I confirmed no more instances of gender are in the code base

**Pull request merged**